CHARTER.adoc

```diff
@@ -0,0 +1,203 @@
+* Multi-dimensional optimizations (spatial chunking scheme, temporal chunking scheme)
+* Supporting typical Earth observation (EO) products (for example, how to encode multispectral bands)
+* Accessing the symbology of the corresponding data
+
+Part of the effort of this working group is to determine what should be kept as a part of the GeoZarr core requirements, and what aspects of geospatial data should be kept as separate conformance classes, which may or may not be applicable to specific sub-domains of the geospatial community. 
+
+=== Statement of relationship of planned work to the current OGC standards baseline
+As the existing draft GeoZarr metadata utilizes Climate and Forecast (CF) attributes, there is an expected relationship with the OGC CF-netCDF Data Model Extension Standard and the OGC netCDF SWG. If necessary, any adjustments or enhancements required for the GeoZarr specification will be considered as a proposal to improve CF conventions as well.
+There are strong connections to other OGC raster / array container standards, specifically NetCDF, HDF5, and GeoTiff. The GeoZarr SWG should seek to leverage existing metadata conventions wherever possible.
+
+As a chunked storage format, there is potentially a strong connection to the `OGC API - Tiles` standard. Map tiles could essentially be mapped 1:1 to an appropriately defined multiscale Zarr array.
+
+=== What is Out of Scope?
+In early conversations around creating a draft GeoZarr specification, concerns arose multiple times around the CF encoding of CRS which may pose issues, see https://github.com/zarr-developers/geozarr-spec/issues/20. While these concerns will be discussed and suggestions created for potentially updating CF conventions, if resolutions cannot be made with the GeoZarr specification, we consider out of scope waiting on any subsequent updates to CF conventions to reflect these suggestions. 
+
+=== Specific Existing Work Used as Starting Point
+* GeoZarr draft specification: https://github.com/zarr-developers/geozarr-spec/ 
+
+=== Is This a Persistent SWG
+
+[x] YES
+
+[ ] NO
+
+=== When can the SWG be Inactivated
+
+The SWG can be inactivated once the SWG identifies no new tasks for the SWG and there are no open Change Requests.
+
+== Description of deliverables
+The GeoZarr SWG will deliver a candidate Standard and associated developer resources.
+
+The SWG expects to have a candidate Standard ready for OGC Architecture Board (OAB) review and public comment within nine months of creation of the SWG. Because example implementations will be developed at the same time the candidate Standard is formalized, reference implementations that fully use GeoZarr should be documented at the same time the candidate Standard goes to vote.
+
+=== Initial Deliverables
+
+The following deliverables will be the initial results of work of the SWG.
+
+* OGC GeoZarr Standard
+
+* GeoZarr developer resources
+
+The targeted start date for this SWG is 
```

Comment:
Is the process close enough to specify a start date? How about "immediately upon approval of the SWG charter"?